sdk/identity/azure-identity/BREAKING_CHANGES.md

```diff
@@ -1,5 +1,17 @@
 # Breaking Changes
 
+## 1.16.0
+
+### Behavioral change to `ManagedIdentityCredential` in IMDS managed identity scenarios
+
+As of `azure-identity` 1.16.0, `ManagedIdentityCredential` makes a minor change to the retry behavior in environments where Instance Metadata Service (IMDS) managed identity is used. The retry backoff factor for IMDS requests was reduced to `0.8` from `2` to prevent excessive delays during retries and to improve responsiveness. This change might affect workloads that rely on the previous retry behavior.
```

Comment:
Please double check with our architects. I don't think it meets the bar of breaking change.

Reply:
After speaking with the Python architects, we've agreed that this change of adjusting the retry backoff factor should not labeled as breaking. Moved to "Other Changes".